rectools/models/sasrec.py

```diff
@@ -575,7 +580,8 @@ def process_dataset_train(self, dataset: Dataset) -> Dataset:
     def _collate_fn_train(
         self,
         batch: List[Tuple[List[int], List[float]]],
-    ) -> Tuple[torch.LongTensor, torch.LongTensor, torch.FloatTensor]:
+        n_negative: int,
```

Comment:
int can't be None, but you are checking it later in the code. Should be `tp.Optional[int] = None`
But you don't need it anyway since you can access it with `self.n_negatives`

Reply:
fixed